src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowEdit.cs

```diff
@@ -58,8 +60,31 @@ protected RowEditBase(long rowId, ResultSet associatedResultSet, IEditTableMetad
         /// </summary>
         public IEditTableMetadata AssociatedObjectMetadata { get; }
 
+        /// <summary>
+        /// Sort ID for a row edit. Ensures that when a collection of RowEditBase objects are
+        /// sorted, the appropriate types are sorted to the top.
+        /// </summary>
+        protected abstract int SortId { get; }
```

Comment:
Eh... I think organizing by what the "thing" is is better -- so since it's a property, it should go in the properties section.